modules/nodes-cluster-overcommit-node-enforcing.adoc

```diff
@@ -63,7 +63,7 @@ spec:
     cpu-cfs-quota: <3>
       - "false"
 ----
-<1> Assign a name to CRD.
+<1> Assign a name to CR.
```

Comment:
We used "CRD" in modules/nodes-nodes-garbage-collection-configuring.adoc
Do we need to make it  consistent?

Reply:
@weinliu Yes. This is a Custom Resource (CR). The discrepancy was caused by my misunderstanding of the distinction between the CRD and CR. From my understanding, the user should not edit the CRD.